fire/docstrings.py

```diff
@@ -410,6 +419,83 @@ def _consume_google_args_line(line_info, state):
   else:
     if state.current_arg:
       state.current_arg.description.lines.append(split_line[0])
+      state.max_line_length = max(state.max_line_length, len(line_info.line))
+      if line_info.previous.line == state.line1: # check for line2
```

Comment:
This uses string equality which can mistakenly detect a non-first line as being line 1 if the same line is repeated later. Using line indexes rather than line contents would be more robust.

Reply:
Understood. My question is how can we use line indexes rather than line_info? Since most of the code consumes the docstring line by line. 

Cmiiw, the check is only done when the line does not have an arg name. Line1 should be a line with the argname. Do you think that line1 (that has arg name in it) would be repeated in any case?

Reply:
One way to get the line index would be to add an additional field to line_info, starting by passing the index in as an argument to _create_line_info at line 191.

> Do you think that line1 (that has arg name in it) would be repeated in any case?

I think it would be exceptionally rare, but want to write robust code as much as possible anyway. One reason for doing this is so that the code is more likely to stay correct when future changes are made.

Reply:
I have added the index as you guided and updated the code to compare indexes instead of strings. Please review the most recent commit.